### PR TITLE
Update turkish.txt

### PR DESCRIPTION
**Pull request by eghwand**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Review by def- — APPROVED** [review]

File: data/languages/turkish.txt [hunk 1]
```diff
@@ -1681,146 +1682,146 @@ Render cut to video
 == Kesiti videoya dönüştür
 
 Error playing demo
-== 
+== Demo oynatırken hata oluştu
 
 Failed saving the replay!
-== 
+== Failed saving the replay!
```
Comment (Learath2) [inline data/languages/turkish.txt:1688]: This one is missing, though idk how I'd translate this either.

File: data/languages/turkish.txt [hunk 2]
```diff
@@ -1681,146 +1682,146 @@ Render cut to video
...
-== 
+== Satır %d - %d (%s)
 
 Locked
-== 
+== Locked
 
 Following
-== 
+== Following
```
Comment (Learath2) [inline data/languages/turkish.txt:1721]: Not really great, idk, these concepts don't really translate well to Turkish.
```suggestion
== Takipte
```

File: data/languages/turkish.txt [hunk 3]
```diff
@@ -1681,146 +1682,146 @@ Render cut to video
...
-== 
+== Sonuç yok
 
 Lines %d - %d (%s)
-== 
+== Satır %d - %d (%s)
 
 Locked
-== 
+== Locked
```
Comment (Learath2) [inline data/languages/turkish.txt:1718]:
```suggestion
== Sabit
```

File: data/languages/turkish.txt [hunk 4]
```diff
@@ -1681,146 +1682,146 @@ Render cut to video
...
-== 
+== Sadece iyileştirmeleri kaydet
 
 Regular background color
-== 
+== Normal arka plan rengi
 
 Entities background color
-== 
+== Varlık arka plan rengi
```
Comment (Learath2) [inline data/languages/turkish.txt:1827]: Yapı demisiz diger cevirilerde.
```suggestion
== Yapı arka plan rengi
```

**Pull request merged**